### PR TITLE
Add Flotiq connector

### PR DESCRIPTION
**Pull request by andrzejwp**
This PR adds the ability to integrate Logic Apps with the [Flotiq headless CMS](https://flotiq.com).

### CODE REVIEW

File: certified-connectors/Flotiq/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,216 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Flotiq headless CMS",
+    "description": "Flotiq headless content management system",
```
Comment (sriyen-msft) [inline certified-connectors/Flotiq/apiDefinition.swagger.json:6]: @andrzejwp Can you please provide the description value more in detailed. It should describe concisely the main purpose and value offered by your connector.

Documentation - [Description](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#description)

File: certified-connectors/Flotiq/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+            "schema": {
+              "allOf":[
+              {"type":"object",
+                "properties":{
+                  "id":{"type":"string"}
+                }
+              },
+              {
+              "type":"object",
+
```
Comment (sriyen-msft) [inline certified-connectors/Flotiq/apiDefinition.swagger.json:85]: @andrzejwp Please remove empty line here.

File: certified-connectors/Flotiq/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+                "parameters":{
+                  "contentTypeId": {
+                    "parameterReference": "contentTypeId"
+                  }
+                },
+                "itemValuePath":"schemaDefinition"
+              }
+            }
+            ]
+            },
```
Comment (sriyen-msft) [inline certified-connectors/Flotiq/apiDefinition.swagger.json:97]: @andrzejwp Please fix indentation issues in swagger.

File: certified-connectors/Flotiq/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+              "value-path": "name",
+              "value-title": "label"
+            }
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "description": "Dynamic schema of objects in selected CTD",
+            "schema": {
+              "allOf":[
```
Comment (sriyen-msft) [inline certified-connectors/Flotiq/apiDefinition.swagger.json:77]: @andrzejwp I do not think `allOf` keyword is not supported in Power Automate. Can you please provide more details on this keyword.

File: certified-connectors/Flotiq/apiProperties.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#16A085",
+    "capabilities": [],
+    "stackOwner": "Flotiq",
+    "publisher": "CodeWave LLC"
```
Comment (sriyen-msft) [inline certified-connectors/Flotiq/apiProperties.json:21]: @andrzejwp Why `publisher` is different from `stackOwner` ?

**Review by sriyen-msft — DISMISSED** [review]
@andrzejwp Please find below warnings we noticed in the swagger validation log which are worth to consider.

`The dynamic operation expects a property on the specified path 'schemaDefinition/allOf/1/properties' which should be defined in the successful response of the invoked operation 'GetContentTypeSchema'. ValuePath: /definitions/CTD/x-ms-dynamic-schema/value-path
The dynamic operation expects a property on the specified path 'schemaDefinition/allOf/1' which should be defined in the successful response of the invoked operation 'GetContentTypeSchema'. ValuePath: /definitions/CTD/x-ms-dynamic-properties/itemValuePath
The dynamic operation expects a property on the specified path 'schemaDefinition' which should be defined in the successful response of the invoked operation 'GetContentTypeSchema'. ValuePath: /paths/~1api~1v1~1content~1{contentTypeId}/post/parameters/1/schema/allOf/1/x-ms-dynamic-properties/itemValuePath
The dynamic operation expects an array on the specified path 'data' which should be defined in the successful response of the invoked operation 'GetCTD'. ValuePath: /paths/~1api~1v1~1content~1{contentTypeId}/post/parameters/0/x-ms-dynamic-values/value-collection`

File: certified-connectors/Flotiq/apiProperties.json (hunk 5 above)
Comment (andrzejwp) [reply certified-connectors/Flotiq/apiProperties.json:21]: CodeWave is the tech team working on Flotiq - the product, which is a separate entity.

File: certified-connectors/Flotiq/apiDefinition.swagger.json (hunk 1 above)
Comment (andrzejwp) [reply certified-connectors/Flotiq/apiDefinition.swagger.json]: Fixed.

File: certified-connectors/Flotiq/apiDefinition.swagger.json (hunk 2 above)
Comment (andrzejwp) [reply certified-connectors/Flotiq/apiDefinition.swagger.json]: Fixed.

File: certified-connectors/Flotiq/apiDefinition.swagger.json (hunk 3 above)
Comment (andrzejwp) [reply certified-connectors/Flotiq/apiDefinition.swagger.json:87]: Fixed.

File: certified-connectors/Flotiq/apiDefinition.swagger.json (hunk 4 above)
Comment (andrzejwp) [reply certified-connectors/Flotiq/apiDefinition.swagger.json]: True, it's not supported. It doesn't break anything, but it doesn't add any value, either. Removed.

**Comment by andrzejwp** [comment]
@sriyen-msft thanks for the comments and suggestions. I addressed them in the recent commit. Regarding the warnings - they are, unfortunately, impossible to fix at this point because:

1. Power platform does not support OpenAPI composition (`allOf` keyword)
2. Flotiq uses composition in the Content Type schema definition

so we will have to live with this until either of these gets resolved ;)

**Review by dougelliott — APPROVED** [review]

**Pull request merged**